README.md

```diff
@@ -1,6 +1,26 @@
-# How to Git Better at Git
+# How to Not Fail Software Engineering by Gitting Better at Git
 
-- Daric Teske
+- Daric Teske (obviously you mean Gauri Ramesh)
+
+| table | with | emojis |
+| ----- | ---- | -----  |
+| `🍞`  | `🥖` | `🍌`  |
+| `🌔`  | `👃` | `🦀`  |
+| `🥔`  | `⭕` | `🎉`  |
```

Comment:
This is the coolest table I've ever seen

Reply:
I like the nose in the middle